adagucserverEC/CDataSource.h

```diff
@@ -373,6 +373,8 @@ class CDataSource {
    * @return 0 on succes, 1 on failure
    */
   int readVariableDataForCDFDims(CDF::Variable *variableToRead, CDFType dataTypeToReturnData);
+
+  std::string getDataSetName();
```

Comment:
Use this in `LayerMetadataStore`? It is now not used.

Reply:
It is now used